web/src/lib/taxation/helpers.ts

```diff
@@ -0,0 +1,58 @@
+import { postTaxFilingsOnboarding } from "@/lib/api-client/apiClient";
+import { UpdateQueue } from "@/lib/types/types";
+import { getCurrentBusiness } from "@businessnjgovnavigator/shared/index";
+import { LookupLegalStructureById } from "@businessnjgovnavigator/shared/legalStructure";
+import { ProfileData } from "@businessnjgovnavigator/shared/profileData";
+const displayBusinessName = (profileData: ProfileData): boolean => {
+  return LookupLegalStructureById(profileData.legalStructureId).elementsToDisplay.has("businessName");
+};
+const displayResponsibleOwnerName = (profileData: ProfileData): boolean => {
+  return LookupLegalStructureById(profileData.legalStructureId).elementsToDisplay.has("responsibleOwnerName");
+};
+interface Props {
+  updateQueue: UpdateQueue;
+  optionalProfileDataIfSeparateContext?: ProfileData;
+}
+// the aim of this fn is to hit postTaxFilingsOnboarding and staged update queue
+export const toBeNamed = async (props: Props) => {
```

Comment:
[boulder] Delete this comment and rename the function.

A better name for this function might more accurately capture what the purpose of the function is without the need of a comment.

Reply:
[boulder] It also looks like this function is missing a return type.

My IDE called that one out for me. If yours isn't, we can look into that. We've got a bunch of default settings and plugins for the workspace that might be helpful.